cmd/tree-walk_test.go

```diff
@@ -479,6 +505,15 @@ func TestSortedness(t *testing.T) {
 	isLeaf := func(volume, prefix string) bool {
 		return !hasSuffix(prefix, slashSeparator)
 	}
+
+	isLeafDir := func(volume, prefix string) bool {
+		entries, listErr := disk1.ListDir(volume, prefix)
```

Comment:
List dir would list all entries perhaps we should list only 1 entry?

Reply:
> List dir would list all entries perhaps we should list only 1 entry?

Done